packages/frontend/src/components/SimpleTable/index.tsx

```diff
@@ -0,0 +1,118 @@
+import { Button, Colors, HTMLTable } from '@blueprintjs/core';
+import { friendlyName } from 'common';
+import React, { FC, useEffect, useState } from 'react';
+import { TrackSection } from '../../providers/TrackingProvider';
+import { SectionName } from '../../types/Events';
+import { TableHeader, TableInnerWrapper, TableWrapper } from './SimpleTable';
+
+interface Props {
+    data: any;
+}
+const SimpleTable: FC<Props> = ({ data }) => {
+    const [paginatedData, setPaginatedData] = useState(data.slice(0, 25));
```

Comment:
If we need pagination we can use `react-table`

Otherwise perhaps we can start by hard coding it to a max of 25 rows or something. What do you think? @TuringLovesDeathMetal do you think that's a useful start. 

Eventually we'll have some configuration for this viz type and the user should be able to configure that.